tests/requirements.txt

```diff
@@ -1,6 +1,7 @@
 backoff==1.10.0
 certifi==2019.3.9
 chardet==3.0.4
+cryptography==2.8
```

Comment:
Why is this added?

Reply:
This is a test requirement and if you don't specify this, Python 3.4 will break.